### PR TITLE
#1456 Literature API summary

### PR DESCRIPTION
**Pull request by mike-podolskiy90**

### CODE REVIEW

**Comment by mike-podolskiy90** [comment]
Deployed to UAT

**Comment by timrobertson100** [comment]
I don’t see it on https://www.gbif-uat.org/developer/summary

**Comment by mike-podolskiy90** [comment]
Hmm, that's strange, I checked.. ok, I'm redeploying

**Comment by mike-podolskiy90** [comment]
done

File: app/views/pages/developer/literature.nunjucks [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                </div>
+            </div>
+        </section>
+
+        {$ api.apiDocTabs('literature', __) $}
+
+        <section id="overview" class="horizontal-stripe light-background">
+            <div class="container">
+                <p>
+                    This API is used to retrieve literature.
```
Comment (timrobertson100) [inline app/views/pages/developer/literature.nunjucks:34]: Suggest change this to:

> This API enables you to search for literature indexed by GBIF, including peer-reviewed papers, citing GBIF datasets and downloads. It powers the <a href="https://www.gbif.org/resource/search?contentType=literature">Literature search</a> on GBIF.org.

**Comment by timrobertson100** [comment]
Thanks, @mike-podolskiy90 

The content LGTM @MortenHofft I'll let you merge since I don't know the codebase

File: app/views/pages/developer/literature.nunjucks (hunk 1 above)
Comment (mike-podolskiy90) [reply app/views/pages/developer/literature.nunjucks]: Thank you, I changed that. I deployed an updated version

**Pull request merged**